internal/cli/opsmanager/admin/backup/s3/update.go

```diff
@@ -0,0 +1,139 @@
+		blockstore.PathStyleAccessEnabled = &opts.pathStyleAccessEnabled
+	}
+
+	return blockstore
+}
+
+// mongocli ops-manager admin backup blockstore(s) update [--assignment][--encryptedCredentials][--id id]
+// [--label label][--loadFactor loadFactor][--uri uri][--ssl][--writeConcern writeConcern] [--awsAccessKey awsAccessKey]
+// [--awsSecretKey awsSecretKey] [--s3AuthMethod s3AuthMethod] [--s3BucketEndpoint s3BucketEndpoint] [--s3BucketName s3BucketName]
+// [--s3MaxConnections s3MaxConnections] [--disableProxyS3 disableProxyS3] [--acceptedTos acceptedTos] [--sseEnabled sseEnabled]
+// [--pathStyleAccessEnabled pathStyleAccessEnabled]
+
+func UpdateBuilder() *cobra.Command {
+	opts := &UpdateOpts{}
+	opts.Template = updateTemplate
+	cmd := &cobra.Command{
+		Use:   "update <blockstoreID>",
+		Args:  cobra.ExactArgs(1),
+		Short: update,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			opts.OutWriter = cmd.OutOrStdout()
+			return opts.init()
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.ID = args[0]
+			return opts.Run()
+		},
+	}
+
+	cmd.Flags().StringVar(&opts.ID, flag.ID, "", usage.BlockstoreID)
+	cmd.Flags().StringSliceVar(&opts.Label, flag.Label, []string{}, usage.Label)
+	cmd.Flags().BoolVar(&opts.Assignment, flag.Assignment, false, usage.Assignment)
+	cmd.Flags().BoolVar(&opts.EncryptedCredentials, flag.EncryptedCredentials, false, usage.EncryptedCredentials)
+	cmd.Flags().Int64Var(&opts.LoadFactor, flag.LoadFactor, 0, usage.LoadFactor)
+	cmd.Flags().StringVar(&opts.URI, flag.URI, "", usage.BlockstoreURI)
+	cmd.Flags().StringVar(&opts.WriteConcern, flag.WriteConcern, "", usage.WriteConcern)
+	cmd.Flags().StringVar(&opts.awsAccessKey, flag.AWSAccessKey, "", usage.AWSAccessKey)
+	cmd.Flags().StringVar(&opts.awsSecretKey, flag.AWSSecretKey, "", usage.AWSSecretKey)
+	cmd.Flags().StringVar(&opts.s3AuthMethod, flag.S3AuthMethod, "", usage.S3AuthMethod)
+	cmd.Flags().StringVar(&opts.s3BucketEndpoint, flag.S3BucketEndpoint, "", usage.S3BucketEndpoint)
+	cmd.Flags().StringVar(&opts.s3BucketName, flag.S3BucketName, "", usage.S3BucketName)
+	cmd.Flags().Int64Var(&opts.s3MaxConnections, flag.S3MaxConnections, 0, usage.S3MaxConnections)
+	cmd.Flags().BoolVar(&opts.disableProxyS3, flag.DisableProxyS3, false, usage.DisableProxyS3)
+	cmd.Flags().BoolVar(&opts.acceptedTos, flag.AcceptedTos, false, usage.AcceptedTos)
+	cmd.Flags().BoolVar(&opts.sseEnabled, flag.SSEEnabled, false, usage.SSEEnabled)
+	cmd.Flags().BoolVar(&opts.pathStyleAccessEnabled, flag.PathStyleAccessEnabled, false, usage.PathStyleAccessEnabled)
+
+	cmd.Flags().StringVarP(&opts.Output, flag.Output, flag.OutputShort, "", usage.FormatOut)
+
+	_ = cmd.MarkFlagRequired(flag.ID)
+	_ = cmd.MarkFlagRequired(flag.URI)
+	_ = cmd.MarkFlagRequired(flag.AcceptedTos)
+	_ = cmd.MarkFlagRequired(flag.S3BucketName)
+	_ = cmd.MarkFlagRequired(flag.S3BucketEndpoint)
+	_ = cmd.MarkFlagRequired(flag.S3AuthMethod)
+	_ = cmd.MarkFlagRequired(flag.SSEEnabled)
```

Comment:
These are all required :/

Reply:
`SSEEnabled` is a boolean, is either on or off so it's always there, unless is like `AcceptedTos` that needs to be on always